quit/web/static/css/bootstrap-theme.min.css

```diff
@@ -0,0 +1,6 @@
+/*!
```

Comment:
Maybe we should not include a static version of bootstrap in the repo, but use something like this: https://pythonhosted.org/Flask-Bootstrap/